docs/scaling/SYSTEM.md

```diff
@@ -150,7 +150,7 @@ ServerlessServices are created from PodAutoscalers.
 ![scale-up-down](images/scale-up-down.png)
 
 At steady state, the autoscaler is constantly scraping the currently active
-revision pods to adjust the scale of the revision constantly. As requests flow
+revision's pods to adjust the scale of the revision constantly. As requests flow
```

Comment:
"revision pods" is actually supposed to be correct here. We don't talk about a specific revision.

Reply:
Got it, thanks for your explanation.